nautobot/extras/tests/test_views.py

```diff
@@ -988,28 +1036,234 @@ def setUpTestData(cls):
             approval_required=True,
             start_time=datetime.now(),
         )
+        ScheduledJob.objects.create(
+            name="test4",
+            task="nautobot.extras.jobs.scheduled_job_handler",
+            job_class="-",
+            interval=JobExecutionType.TYPE_IMMEDIATELY,
+            user=user,
+            approval_required=False,
+            start_time=datetime.now(),
+        )
+
+    def test_only_approvable_is_listed(self):
+        self.add_permissions("extras.view_scheduledjob")
+
+        response = self.client.get(self._get_url("list"))
+        self.assertHttpStatus(response, 200)
+        self.assertNotIn("test4", extract_page_body(response.content.decode(response.charset)))
 
 
 class JobResultTestCase(
+    TestJobMixin,
+    ViewTestCases.GetObjectViewTestCase,
     ViewTestCases.ListObjectsViewTestCase,
+    ViewTestCases.DeleteObjectViewTestCase,
+    ViewTestCases.BulkDeleteObjectsViewTestCase,
 ):
     model = JobResult
 
     @classmethod
     def setUpTestData(cls):
         obj_type = ContentType.objects.get(app_label="extras", model="job")
         JobResult.objects.create(
-            name="test1",
+            name="local/test_view/TestJob",
             job_id=uuid.uuid4(),
             obj_type=obj_type,
         )
         JobResult.objects.create(
-            name="test2",
+            name="local/test_view/TestJob2",
             job_id=uuid.uuid4(),
             obj_type=obj_type,
         )
         JobResult.objects.create(
-            name="test3",
+            name="local/test_view/TestJob3",
             job_id=uuid.uuid4(),
             obj_type=obj_type,
         )
+
+
+class JobTestCase(
+    TestCase,
```

Comment:
Perhaps add a comment or docstring explaining why this isn't using any of the generic base classes? (Because Job isn't a proper database model, right?)